### ISSUE TITLE
Brick status value for "Gluster-at-a-glance"

### ISSUE THREAD

**Issue by r0h4n** [Tendrl/monitoring-integration#365]
Attached screen shot, "Volume status"  "Host status"  show "status" column as 0 for active

But the Brick status has "status" column as 1 for active. 

Any reason for this different behavior with brick statuses ?

![screenshot from 2018-03-07 01-16-06](https://user-images.githubusercontent.com/369941/37054763-bc9e82bc-21a5-11e8-9516-c0f1f4a8c39c.png)

**Comment by cloudbehl** [comment]
@r0h4n There is logic written in grafana for coloring the panel based upon these values that's why the different values are assigned to a different status.

volume status and node status comes as "UP" and "Down" whereas the brick status comes as "Started" and "Stopped".

Status Mapper[1] for grafana:

[1]. https://github.com/Tendrl/monitoring-integration/blob/master/tendrl/monitoring_integration/graphite/__init__.py#L566

**Comment by r0h4n** [comment]
There is scope to unify these statuses (up, started, down, stopped).  @nthomas-redhat  thoughts?

**Comment by nthomas-redhat** [comment]
@cloudbehl , I agree with @r0h4n . This should be on the similar lines, otherwise its going to be really confusing. We need to take it up for this release.
